src/Servant/API/Routes.hs

```diff
@@ -282,6 +284,14 @@ printRoutes = traverse_ printRoute $ getRoutes @api
   where
     printRoute = T.putStrLn . renderRoute
 
+{- | Get all the routes of an API, sort them by path and method, and print them to stdout.
+ See 'renderRoute' for examples.
+-}
+printRoutesSorted :: forall api. HasRoutes api => IO ()
+printRoutesSorted = traverse_ printRoute . sort $ getRoutes @api
+  where
+    printRoute = T.putStrLn . renderRoute
```

Comment:
Maybe one glitch:
```
DELETE /someRoute
GET /someRoute
POST /someRoute
```
Maybe padding a bit would look even nicer?
```
DELETE /someRoute
GET    /someRoute
POST   /someRoute
```

Reply:
It does look nicer, but it comes with a (slight) performance impact: since a `Method` is just a `ByteString`, in order to guarantee that the paths lined up we'd have to traverse the whole list of `Route`s to find the longest `Method` so we know how much to pad them.

Reply:
That being said, there is a finite pre-defined set of valid HTTP methods, so we could just pad them all to the width of the longest (`CONNECT`/`OPTIONS`).
Still, I'd rather leave that to a separate PR since it's an orthogonal issue.

Reply:
OK, no problem with doing that in a separate issue. We're sorting by `Method` already, I think it already implies traversing all `ByteString`s.